paasta_itests/steps/paasta_serviceinit_steps.py

```diff
@@ -110,6 +110,22 @@ def chronos_status_verbose_returns_healthy(context, service_instance):
     assert "Running Tasks:" in output
 
 
+@then((u'paasta_serviceinit status -vv for the service_instance "{service_instance}"'
+       ' exits with return code 0 and the correct output'))
+def paasta_serviceinit_tail_stdstreams(context, service_instance):
+    cmd = "../paasta_tools/paasta_serviceinit.py --soa-dir %s %s status -vv" % (context.soa_dir, service_instance)
+    print 'Running cmd %s' % cmd
+    (exit_code, output) = _run(cmd)
```

Comment:
> remove the surrounding brackets from (exit_code, output)

Removed the parens there, and anywhere else we were doing the same thing in the itests.